examples/MicroXSReader.ipynb

```diff
@@ -0,0 +1,629 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) 2017-2018 [Serpent-Tools developers team](https://github.com/CORE-GATECH-GROUP/serpent-tools/graphs/contributor), GTRC\n",
```

Comment:
Link is broken. Add an `s` to the end of the link - https://github.com/CORE-GATECH-GROUP/serpent-tools/graphs/contributors